test/functional/wallet_resendwallettransactions.py

```diff
@@ -70,6 +70,7 @@ def run_test(self):
             # Tell scheduler to call MaybeResendWalletTxn now.
             node.mockscheduler(1)
         # Give some time for trickle to occur
+        time.sleep(1)
```

Comment:
in commit bbc01cf8b2965f716e899f413d36e2beb48c1f06: what is this?

Reply:
This was originally in d7fad1690c27bf270774da6eebc4ef8b550caf09. However, i just ran the test without the line and it seems to not be needed. Removed.

Reply:
No need to keep an empty commit

Reply:
Removed.